tests/behaviour/array_storage_index_zeroed_test_test.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import pytest
+from starkware.starknet.compiler.compile import compile_starknet_files
+from starkware.starknet.testing.state import StarknetState
+from yul.main import transpile_from_solidity
+from yul.starknet_utils import deploy_contract, invoke_method
```

Comment:
We know have a different package structure. All of our modules are placed inside of warp package, so you should prefix imports with `warp.`